specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -506,6 +645,111 @@
       },
       "x-nullable": false
     },
+    "CopyAuthorizationResult": {
+      "description": "Request parameter that contains authorization claims for copy operation.",
+      "required": [
+        "modelId",
+        "accessToken",
+        "expirationDateTimeTicks"
+      ],
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Model identifier.",
+          "type": "string"
+        },
+        "accessToken": {
+          "description": "Token claim used to authorize the request.",
+          "type": "string"
+        },
+        "expirationDateTimeTicks": {
+          "description": "The time when the access token expires. The date is represented as the number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "CopyRequest": {
+      "description": "Request parameter to copy an existing custom model.",
+      "required": [
+        "targetResourceId",
+        "targetResourceRegion",
+        "copyAuthorization"
+      ],
+      "type": "object",
+      "properties": {
+        "targetResourceId": {
+          "description": "Azure Resource Id of the target Form Recognizer resource where the model is copied to.",
+          "maxLength": 1024,
+          "type": "string",
+          "pattern": "^/subscriptions/[^/]*/resourceGroups/[^/]*/providers/Microsoft.CognitiveServices/accounts/[^/]*$",
+          "x-ms-azure-resource": true
+        },
+        "targetResourceRegion": {
+          "description": "Location of the target Azure resource. A valid Azure region name supported by Cognitive Services.",
+          "type": "string",
+          "pattern": "^[a-z0-9]+$",
+          "minLength": 1,
+          "maxLength": 24
+        },
+        "copyAuthorization": {
+          "description": "Entity that encodes claims to authorize the copy request.",
+          "$ref": "#/definitions/CopyAuthorizationResult"
+        }
+      }
+    },
+    "CopyOperationResult": {
+      "description": "Status and result of the queued copy operation.",
+      "type": "object",
+      "required": [
+        "status",
+        "createdDateTime",
+        "lastUpdatedDateTime"
+      ],
+      "properties": {
+        "status": {
+          "description": "Operation status.",
+          "$ref": "#/definitions/OperationStatus"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Date and time (UTC) when the copy operation was submitted.",
+          "type": "string",
+          "x-nullable": false
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Date and time (UTC) when the status was last updated.",
```

Comment:
What does this DateTime signify?  Created is when the copy operation was submitted, is lastUpdated when the copy operation completed?

These are the same DateTime names that are used for starting and completing training of a model.  Do these times have the same semantic meaning?  Would it make sense to give them different names here, such as startCopyDateTime and endCopyDateTime?  Would it make sense to maintain the time the model was trained?  I understand from conversations with Chris Stone that people use the timestamp on the model to know what trained version it was -- it might make sense to retain that information here.

Reply:
This is the "base" contract portion of the result and is the same for train and copy operations.